packages/ag-charts-community/src/chart/update/baseLayoutProcessor.ts

```diff
@@ -135,5 +144,9 @@ export class BaseLayoutProcessor implements UpdateProcessor {
                 caption.node.x = rect.x + rect.width - bbox.width - titlePadding;
             }
         }
+
+        [title, subtitle, footnote].forEach((c) =>
+            c.updateA11yText(this.proxyInteractionService, this.proxyTextContainer)
+        );
```

Comment:
This feels like an odd place to be doing this, as the A11y update doesn't seem related to layout concerns?

Maybe `Chart.updateDOM()` would fit better? (I'm not sure if that is always invoked at the right time though?)

Reply:
This update is related to layout concerns. If the CSS bounds of the text aren't set correctly then the screen-reader will indicate the incorrect bounds.

---

![Screenshot 2024-07-08 at 15 04 32](https://github.com/ag-grid/ag-charts/assets/23700103/7721e6d1-c5f6-486c-a6f8-28d3a54127ae)
_Screenshot on https://github.com/ag-grid/ag-charts/commit/f9cfa7e4998a284adcfd5492202f207b3ce6a178._

---

![Screenshot 2024-07-08 at 15 19 32](https://github.com/ag-grid/ag-charts/assets/23700103/92f41ec2-80fa-4ae4-b526-620ec51042b3)
_Screenshot on https://github.com/ag-grid/ag-charts/commit/17a3e1f2c89512baabd08f36f6d20b9940a93d3b._